includes/wsuwp-shortcode-fundselector.js

```diff
@@ -50,18 +50,20 @@ jQuery(document).ready(function($) {
 		if(categoryName) {
 			
 			var restUrl = wpData.request_url_base + encodeURIComponent(categoryName) + "?per_page=100&orderby=name";
+			var category = $(this).attr("data-name");
+			var description = $(this).attr("data-description");
```

Comment:
If this isn't set for some reason, it should default to "category".
`var description = $(this).attr("data-description") || "category";`